packages/mediacenter/kodi/patches/kodi-100.14-use-alsa-and-pulse-together.patch

```diff
@@ -1,17 +1,18 @@
-From 0bb21ebfe5f26545ccc9f105560c6c0fcbce77b3 Mon Sep 17 00:00:00 2001
-From: MilhouseVH <milhouseVH.github@nmacleod.com>
-Date: Wed, 26 Sep 2018 02:47:03 +0100
-Subject: [PATCH] allow using alsa and pulse together
+From 2e7de29a69b37ad0a7d8632c5aafcdd02a4f550d Mon Sep 17 00:00:00 2001
+From: 5schatten <supervisedthinking@gmail.com>
+Date: Tue, 7 May 2019 01:30:10 +0200
+Subject: [PATCH] [PATCH] allow using alsa and pulse together
```

Comment:
`Subject: [PATCH] allow using alsa and pulse together` :)

Reply:
fixed